src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.annotation;
+
+import com.microsoft.azure.functions.BrokerAuthenticationMode;
+import com.microsoft.azure.functions.BrokerProtocol;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>Annotation for Kafka output bindings</p>
+ */
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+@CustomBinding(direction = "out", name = "kafkaOutput", type = "kafka")
```

Comment:
We may need to remove this as it is already merged to plugin.

Reply:
Done!